tinygrad/renderer/cstyle.py

```diff
@@ -237,6 +240,8 @@ class MetalLanguage(CStyleLanguage):
   gid = [f"gid.{chr(120+i)}" for i in range(3)]
   lid = [f"lid.{chr(120+i)}" for i in range(3)]
   extra_args = ['uint3 gid [[threadgroup_position_in_grid]]', 'uint3 lid [[thread_position_in_threadgroup]]']
+  def render_cast(self, x: List[str], var_dtype: DType, bitcast=False) -> str:
+      return f"as_type<{var_dtype.name}>({x[0]})" if bitcast else super().render_cast(x, var_dtype)
```

Comment:
spacing is wrong here, how does linter pass?

Reply:
I'll fix and check linter